docs/cis_self_assessment.md

```diff
@@ -352,176 +349,97 @@ root:root
-</details>
-
-**Result:** Not Applicable
+```bash
+stat -c %a /var/lib/rancher/rke2/server/cred/controller.kubeconfig
+644
+```
 
 **Remediation:**
-RKE doesn’t require or maintain a configuration file for the controller manager. All configuration is passed in as arguments at container run time.
-
+By default, RKE2 creates this file at `/var/lib/rancher/rke2/server/cred/controller.kubeconfig` and automatically sets its permissions to `644`. No manual remediation needed.
 
 #### 1.1.18
-Ensure that the `controller-manager.conf` file ownership is set to `root:root` (Scored)
+Ensure that the `controller.kubeconfig` file ownership is set to `root:root` (Scored)
 <details>
 <summary>Rationale</summary>
-The controller-manager.conf file is the kubeconfig file for the Controller Manager. You should set its file ownership to maintain the integrity of the file. The file should be owned by root:root.
+The controller.kubeconfig file is the kubeconfig file for the Scheduler. You should set its file ownership to maintain the integrity of the file. The file should be owned by root:root.
+
+In RKE2, this file is located at `/var/lib/rancher/rke2/server/cred/controller.kubeconfig`.
 </details>
 
-**Result:** Not Applicable
+**Result:** Pass
 
-**Remediation:**
-RKE doesn’t require or maintain a configuration file for the controller manager. All configuration is passed in as arguments at container run time.
+**Audit:**
+```bash
+stat -c %U:%G /var/lib/rancher/rke2/server/cred/controller.kubeconfig
+root:root
+```
 
+**Remediation:**
+By default, RKE2 creates this file at `/var/lib/rancher/rke2/server/cred/controller.kubeconfig` and automatically sets its ownership to `root:root`.
 
 #### 1.1.19
 Ensure that the Kubernetes PKI directory and file ownership is set to `root:root` (Scored)
 <details>
 <summary>Rationale</summary>
-Kubernetes makes use of a number of certificates as part of its operation. You should set the ownership of the directory containing the PKI information and all files in that directory to maintain their integrity. The directory and files should be owned by root:root.
+Kubernetes makes use of a number of certificates as part of its operation. You should set the ownership of the directory containing the PKI information and all files in that directory to maintain their integrity. The directory and files should be owned by root:root. 
 </details>
 
 **Result:** Pass
 
-**Remediation:**
-Run the below command (based on the file location on your system) on the master node.
-For example,
-
-``` bash
-chown -R root:root /etc/kubernetes/ssl
-```
-
 **Audit:**
-
-```
-stat -c %U:%G /etc/kubernetes/ssl
+```bash
+stat -c %U:%G /var/lib/rancher/rke2/server/tls
+root:root
 ```
 
-**Expected result**:
+**Remediation:**
+By default, RKE2 creates the directory and files with the expected ownership of `root:root`. No manual remediation should be necessary.
 
-```
-'root:root' is present
+```bash
```

Comment:
You still included the chown in the remediation. I just want to drop any manual remediation completely from the doc. Should only need the above statement.